src/marshmallow/context.py

```diff
@@ -0,0 +1,25 @@
+"""Objects related to serializtion/deserialization context"""
+
+import contextlib
+import contextvars
+
+from marshmallow.utils import no_default
+
+
+class Context(contextlib.AbstractContextManager):
+    _current_context: contextvars.ContextVar = contextvars.ContextVar("context")
+
+    def __init__(self, context):
+        self.context = context
+
+    def __enter__(self):
+        self.token = self._current_context.set(self.context)
+
+    def __exit__(self, *args, **kwargs):
+        self._current_context.reset(self.token)
+
+    @classmethod
+    def get(cls, default=no_default):
```

Comment:
suggestion: could use `Ellipsis`

```suggestion
    def get(cls, default=Ellipsis):
```

i encountered similar usage in structlog: https://github.com/hynek/structlog/blob/e51f34705d0e6c6551ca0915eded697dc99f0e57/src/structlog/contextvars.py#L133

Reply:
Nice! I used the `...` form, which looks equivalent.